### ISSUE TITLE
Cosmos72 do you want some .debs?

### ISSUE THREAD

**Issue by jafcobend** [cosmos72/twin#73]
Hey @cosmos72 do you want some Debian packages for TWIN? I still need to patch TWIN to put the system.* config files in /etc (instead of /usr/lib/twin) but I build them for i386, amd64, armhf, arm64 and RISC-V. I have broken it up into 3 pieces: core + terminal support + clients, X11 support and development files.

I'm still refining the packages but I figured since you jumped on my other issues so quickly I just had to offer them to the project. I use the long abandoned `yada` tool to make them, since its the only Debian build system that's sane. I mention that because my packaging files won't be accepted by Debian officially. But the finished goods work on Debian and derived distros.

**Comment by cosmos72** [comment]
Thanks for the offer!

As you point out, these cannot become official Debian packages, which makes them definitely less than optimal:

it means any user installing them would need to trust they are not compromised - i.e. not only trust the source code, but also who compiled them and the *whole* system where they have been compiled.

**Comment by Mirppc** [comment]
The other issue is the twin packages that the debian project maintained are wayyyy out of date.  when i looked into it they where several version behind when the project was only on sourceforge.  So unless someone with the debian project is willing to maintain it and do the QA it might not be a great idea.


However there are build systems like the OpenBuildService that might make QA and building for multiple distros.  Just a thought.

TLDR though is i agree with @cosmos72 .
